src/main/java/calculator/model/Calculator.java

```diff
@@ -0,0 +1,34 @@
+package calculator.model;
+
+import java.util.List;
+
+public class Calculator {
+
+    private double result;
+
+    public double calculatePlusNumber(List<String> numbers) {
+        for (String num : numbers) {
+            validator(num);
+            result += Double.parseDouble(num);
+        }
+        return result;
+    }
+
+    public Object determine_type(double result) {
```

Comment:
```suggestion
    public Object determineType(double result) {
```
저는 자바에서 주로 사용하는 카멜 케이스에 맞춰서 이렇게 작성했을 것 같아요! 혹시 스네이크 케이스로 사용하신 다른 이유가 있으실까요??

Reply:
아이고 실수했습니다,,, 다음부터는 주의해야겠군요